### PR TITLE
64 adds a utils section to contributing.md

### LINKED ISSUE #64: Documentation Content Completeness

### PR DESCRIPTION
**Pull request by wyli**
Fixes #64.

### Description
documentation updates

### Status
**Ready**

### Types of changes
- [x] Docstrings/Documentation updated

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,6 +61,21 @@ License information: all source code files should start with this paragraph:
...
+MONAI provides a set of generic utility functions and frequently used routines.
+These are located in [``monai/utils``](./monai/utils/) and in the module folders such as [``networks/utils.py``](./monai/networks/).
+Users are encouraged to use these common routines to improve code readability and reduce the code maintenance burdens.
+
+Notably,
+- ``monai.module.export`` decorator can make the module name shorter when importing,
+for example, ``import monai.transforms.Spacing`` is the equivalent of ``monai.transforms.transforms.Spacing``, since
+``class Spacing`` is decorated with ``@export("monai.transforms")``.
+
+- ``monai.utils.aliases.alias`` decorator can make alternative class name,
```
Comment (ericspod) [inline CONTRIBUTING.md]: This section on alias is not correctly describing it. What `alias` does is add the alias name to the global variable GlobalAliases which is used by `resolve_name` to relate an alias or actual name to a type. This was intended for use with yaml or other script files in a similar way to Catalyst which hasn't made it into MONAI yet. I would leave the aliasing in the codebase as we'll want it later but I would change this section to state that `resolve_name` must be used to resolve an alias name to the real type. The alternative to consider is actually duplicating the decorated definition in the containing module under the alias name, eg. add `RandomZoomDict` as an actual member of module which is just a reference to `RandZoomd`.

Comment (wyli) [reply CONTRIBUTING.md]: i thought these are for aliases like this: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/layers/pooling.py#L38  

this is confusing. I'll remove all the `alias` usage in another PR

Comment (ericspod) [reply CONTRIBUTING.md]: This is what I'd say we could convert the alias functionality into. I would leave in the alias decorators that are there now for future use.

Comment (wyli) [reply CONTRIBUTING.md]: sure we update the docstring to make it clear that it's for future use

**Pull request merged**